src/ripple/app/tx/impl/SetHook.cpp

```diff
@@ -205,7 +205,7 @@ validateHookParams(SetHookCtx& ctx, STArray const& hookParams)
 // infer which operation the user is attempting to execute from the present and
 // absent fields
 HookSetOperation
-SetHook::inferOperation(STObject const& hookSetObj)
+SetHook::inferOperation(SetHookCtx& ctx, STObject const& hookSetObj)
```

Comment:
why was this parameter added? it's not used?

Reply:
I used to check the temDISABLED field for the HookCanEmit field.
I'll delete it because it’s checked outside inferOperation now.